api_tests/nodes/views/test_node_contributors_list.py

```diff
@@ -1006,40 +1006,58 @@ def setUp(self):
         self.private_url = '/{}nodes/{}/contributors/'.format(API_BASE, self.private_project._id)
         self.public_url = '/{}nodes/{}/contributors/'.format(API_BASE, self.public_project._id)
 
-        self.payload_one = {
-                'id': self.user_two._id,
-                'type': 'contributors',
-                'attributes': {
-                    'bibliographic': True,
-                    'permission': "admin"
-                }
+        self.payload_one_public = {
+            'id': make_contrib_id(self.public_project._id, self.user_two._id),
+            'type': 'contributors',
+            'attributes': {
+                'bibliographic': True,
+                'permission': "admin"
+            }
         }
 
-        self.payload_two = {
-                'id': self.user_three._id,
-                'type': 'contributors',
-                'attributes': {
-                    'bibliographic': False,
-                    'permission': "write"
-                }
+        self.payload_one_private = {
+            'id': make_contrib_id(self.private_project._id, self.user_two._id),
+            'type': 'contributors',
+            'attributes': {
+                'bibliographic': True,
+                'permission': "admin"
+            }
+        }
+
+        self.payload_two_public = {
+            'id': make_contrib_id(self.public_project._id, self.user_three._id),
+            'type': 'contributors',
+            'attributes': {
+                'bibliographic': False,
+                'permission': "write"
+            }
+        }
+
+        self.payload_two_private = {
+            'id': make_contrib_id(self.private_project._id, self.user_three._id),
+            'type': 'contributors',
+            'attributes': {
+                'bibliographic': False,
+                'permission': "write"
+            }
         }
 
     def test_bulk_update_contributors_blank_request(self):
         res = self.app.patch_json_api(self.public_url, auth=self.user.auth, expect_errors=True, bulk=True)
         assert_equal(res.status_code, 400)
 
     def test_bulk_update_contributors_dict_instead_of_list(self):
-        res = self.app.put_json_api(self.public_url, {'data': self.payload_one},
+        res = self.app.put_json_api(self.public_url, {'data': self.payload_one_public},
                                     auth=self.user.auth, expect_errors=True, bulk=True)
         assert_equal(res.status_code, 400)
 
     def test_bulk_update_contributors_public_project_one_not_found(self):
         invalid_id = {
-            'id': '12345',
+            'id': '12345-abcde',
```

Comment:
Could you make an additional test wherever this diff occurs that tries it with the non-hyphenated id and ensure that it just fails with a 400-series error and not a 500-series error?